tools/testnet-faucets.md

```diff
@@ -22,7 +27,7 @@ You can request some Sepolia ETH on Ethereum Sepolia testnet and then bridge the
 
 ***
 
-## [Aori Faucet](https://app.aori.io/faucet)
+### [Aori Faucet](https://app.aori.io/faucet)
```

Comment:
Please remove this character to leave the rest of the page the same as it was, only adding the new changes.